build.sbt

```diff
@@ -30,5 +41,4 @@ lazy val settings = Seq(
     val required = "1.8"
     val current  = sys.props("java.specification.version")
     assert(current == required, s"Unsupported JDK: java.specification.version $current != $required")
-  }
-)
+  }) ++ formattingSettings
```

Comment:
The `formattingSettings` aren't applied to the root project.
```
> scalariformPreferences
[info] core/compile:scalariformPreferences
[info]  FormattingPreferences(Map(DanglingCloseParenthesis -> Prevent, DoubleIndentClassDeclaration -> true))
[info] {.}/*:scalariformPreferences
[info]  FormattingPreferences(Map(SpacesAroundMultiImports -> true, DoubleIndentClassDeclaration -> true))
```

Do we care? I don't think so, but figured I'd ask.